pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -121,6 +123,16 @@ func NewAlertmanager(ctx context.Context, orgID int64, cfg *setting.Cfg, store A
 			return stateStore.SaveNotificationLog(context.Background(), state)
 		},
 	}
+	l := log.New("ngalert.notifier.alertmanager", "org", orgID)
+	action := stages.Disabled
+	if featureToggles.IsEnabledGlobally(featuremgmt.FlagAlertingAlertmanagerExtraDedupStage) {
+		if featureToggles.IsEnabledGlobally(featuremgmt.FlagAlertingAlertmanagerExtraDedupStageStopPipeline) {
+			action = stages.StopPipeline
+		} else {
+			action = stages.LogOnly
+		}
+		l.Info("Initializing Alertmanager", "extraDedupStage", action)
```

Comment:
Nit, feel free to ignore, but we mostly use snake case for logger context ([example 1](https://github.com/grafana/grafana/blob/111f9732421eeaf70e52c6573178e73c60265f47/pkg/services/ngalert/api/api_ruler.go#L380-L381), [example 2](https://github.com/grafana/alerting/blob/d49e2e0b7d65f0f2ef4915f74de481bf7506efc6/notify/grafana_alertmanager.go#L681)).
```suggestion
		l.Info("Initializing Alertmanager", "extra_dedup_stage", action)
```